gtsfm/data_association/data_assoc.py

```diff
@@ -162,6 +179,14 @@ def run(
             ],
         )
 
+        if gt_computed_exit_codes_distribution is not None:
+            for exit_code_tuple, count in gt_computed_exit_codes_distribution.items():
+                metric_name = "#tracks triangulated with GT pose: {}, computed pose: {}".format(
```

Comment:
I think it would be good to add a comment explaining what "#tracks triangulated with GT pose: {}, computed pose: {}" means in more complete terms.

Reply:
I feel the metric label will get tol verbose in that case. I can maybe have a seperate table for this, and then add a description above for that table? 

What do y'all think?

Reply:
I just meant as a comment above the code block or something, not in the actual table.

Reply:
Oh ok. Got it. I'll do it

Reply:
Ayush, could you explain in just a sentence or two here in an inline comment explaining what it means?

Also, maybe we should make temp variables that are more readable for `exit_code_tuple[0].name, exit_code_tuple[1].name`, since i cant really tell what that tuple consists of

Reply:
would 
```python
for (track_class, track_exit_code), count in gt_computed_exit_codes_distribution.items():
```
be clearer?

Reply:
It is not exactly the terms you mention, but I get your point. Will update it.